### DISCUSSION TITLE
error conda is not recognized di vs code

### DISCUSSION THREAD

**Discussion by jonathanariel** [teamalgoritma/community#180] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/107382811/177957001-438ddc04-d943-4b27-9dc0-41871886fe2d.png)
permisi, mau nanya ini kenapa ya conda not recognized, padahal interpreternya udah sesuai dengan virtual environment yang dipakai?

**Comment by kevbow** [comment, marked as answer]
Halo @jonathanariel,

Silakan untuk mengganti tipe terminal dari powershell menjadi command prompt seperti gambar berikut:
![image](https://user-images.githubusercontent.com/68848168/177959740-5fe128d9-42eb-441c-84ea-2c5acb3595c0.png)

**Reply by jonathanariel** [reply]
ah iya sudah bisa, terima kasih kak
